src/rust/multi.bato/res/source.json

```diff
@@ -0,0 +1,157 @@
+{
+	"info": {
+		"id": "multi.bato",
+		"lang": "multi",
+		"name": "BatoTo",
+		"version": 1,
+		"url": "https://bato.to",
```

Comment:
you *could* do `urls` instead of `url` and list the mirror urls, but you'd probably have to handle their links then.

Reply:
done

Reply:
handle url works with multiple links